### ISSUE TITLE
Huge app size

### ISSUE THREAD

**Issue by Donkey-Doug** [crackededed/Xtra#323]
The app takes up 5gb in my phone.

Proof:
![Screenshot_20230621-151526_Settings](https://github.com/crackededed/Xtra/assets/50382414/a072f32d-b588-4894-b24c-294c699c1d13)

**Comment by crackededed** [comment]
do you have any videos downloaded? you should tap on xtra on this screen to see what is taking space

**Comment by crackededed** [comment]
show the storage screen for xtra

**Comment by Donkey-Doug** [comment]
![Screenshot_20230628-230327_Settings](https://github.com/crackededed/Xtra/assets/50382414/ab8bcd2c-804b-44c0-aff6-1ddfd0dc64b4)

**Comment by crackededed** [comment]
have you downloaded any videos previously? you may have some files still in the download folder: 
`Android/data/com.github.andreyasadchy.xtra/files/.downloads`
deleting large videos takes some time. if you have exited the app shortly after deleting videos the deletion process may have been interrupted

**Comment by Itsfadixx** [comment]
> have you downloaded any videos previously? you may have some files still in the download folder: `Android/data/com.github.andreyasadchy.xtra/files/.downloads` deleting large videos takes some time. if you have exited the app shortly after deleting videos the deletion process may have been interrupted


__---___------_
Itsfadixx here!




I hope this message finds you well. I recently came across your fork of [xtra] on GitHub and found it to be an intriguing project. After careful consideration, I decided to fork your repository and make some modifications tailored specifically for [Kick.com].

First and foremost, I want to express my appreciation for your work and the effort you put into developing the original project. I understand the value of open-source contributions, and I believe in the collaborative nature of the developer community.

With my modifications, the codebase has been adjusted to suit the requirements of [Kick.com] rather than Twitch. These changes include significant alterations to the internal API code. Before moving forward with any further actions, I wanted to reach out to you to discuss the potential implications from a legal standpoint.

I have thoroughly reviewed the license attached to your fork and its compatibility with the changes I've made. However, I understand the importance of obtaining your consent and ensuring that I am adhering to the proper legal frameworks. Therefore, I kindly request your guidance and clarification on the following matters:

1. The original licensing terms: Could you please confirm the specific open-source license under which your project is distributed? This will help me better understand the rights and restrictions associated with the codebase.

2. Modification and redistribution: Based on my modifications, I am considering publishing my fork as an open-source project tailored for [Kick.com]. I would like to ensure that my actions comply with the original license and meet your expectations as the original author. Are you open to discussing the possibility of me publishing this modified fork? If so, I would appreciate any specific requirements or conditions you may have.

3. Attribution and acknowledgment: I value the importance of giving credit where it is due. If I proceed with publishing this modified fork, I will ensure that appropriate attribution to you as the original author is provided. Additionally, I will clearly highlight the changes made to suit [Kick.com] in the project documentation. Please let me know if you have any preferences regarding attribution or if there are any specific details you would like me to include.

I understand that legal matters can be complex, and I want to approach this situation with transparency and professionalism. If you have any concerns or suggestions, please feel free to share them. I am open to discussing any alternative arrangements or addressing any issues you may have.

Thank you for your time and consideration. I look forward to your response and the opportunity to further discuss this matter. Your guidance and cooperation would be greatly appreciated.

Best regards,

Itsfadixx 
@itsfadixx (discord)

**Comment by crackededed** [comment]
yeah you can fork it

**Comment by Itsfadixx** [comment]
> yeah you can fork it

"meh. Its will remain private"  -Nopbreak, Developer of PurpleTV.

**Comment by crackededed** [comment]
the deletion issue is mentioned in #57 so i'll close this

**Comment by Donkey-Doug** [comment]
> have you downloaded any videos previously? you may have some files still in the download folder: `Android/data/com.github.andreyasadchy.xtra/files/.downloads` deleting large videos takes some time. if you have exited the app shortly after deleting videos the deletion process may have been interrupted

Yeah, I need root to access that. Maybe use a different folder by default for downloads?
